client/src/components/ContactHost.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {
+  Wrapper,
+  StyledButton,
+  Modal,
+  ModalContent,
+  FacebookButton,
+  GoogleButton,
+  EmailButton,
+  Line,
+  Login,
+} from './styles/ContactHost.styles';
+
+export default class ContactHost extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleModalDisplay = this.handleModalDisplay.bind(this);
+    this.handleModalHide = this.handleModalHide.bind(this);
+    this.state = {
+      displayModal: 'none',
+    };
+  }
+
+  handleModalDisplay() {
```

Comment:
You can pass in a variable 'flex' or 'none' here and set state accordingly so you don't have 2 functions that do almost the same thing.